### PR TITLE
[DO NOT MERGE] test CI

### PR DESCRIPTION
**Pull request by jameslamb**
## Description

Testing https://github.com/rapidsai/build-planning/issues/58#issuecomment-2272191453.

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/rmm/blob/HEAD/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.

### CODE REVIEW

**Comment by jakirkham** [comment]
Seeing the following [error on CI]( https://github.com/rapidsai/rmm/actions/runs/10274629189/job/28431769596?pr=1638#step:7:1857 ):

```python
    WARNING: Use cmake.version instead of cmake.minimum-version with scikit-build-core >= 0.8
    ERROR: Cannot set both cmake.minimum_version and cmake.version; use version only for scikit-build-core >= 0.8.
```

Wonder how both `cmake.minimum_version` and `cmake.version` are being set

File: python/librmm/pyproject.toml [hunk 1]
```diff
@@ -15,8 +15,8 @@
 [build-system]
 build-backend = "rapids_build_backend.build"
 requires = [
+    "git+https://github.com/scikit-build/scikit-build-core.git@henryiii/fix/minversionlogic",
```
Comment (henryiii) [inline python/librmm/pyproject.toml:18]:
```suggestion
    "scikit-build-core @ git+https://github.com/scikit-build/scikit-build-core.git@henryiii/fix/minversionlogic",
```

File: python/rmm/pyproject.toml [hunk 2]
```diff
@@ -15,8 +15,8 @@
 [build-system]
 build-backend = "rapids_build_backend.build"
 requires = [
+    "git+https://github.com/scikit-build/scikit-build-core.git@henryiii/fix/minversionlogic",
```
Comment (henryiii) [inline python/rmm/pyproject.toml:18]:
```suggestion
    "scikit-build-core @ git+https://github.com/scikit-build/scikit-build-core.git@henryiii/fix/minversionlogic",
```

File: python/librmm/pyproject.toml (hunk 1 above)
Comment (jameslamb) [reply python/librmm/pyproject.toml]: blegh thank you. We programmatically replace this with the contents from file `dependencies.yaml` at the root of the repo, so I just updated this there and pushed https://github.com/rapidsai/rmm/pull/1638/commits/f6c2b5c4f971eb2a0082409959111cbdbc324aab.

**Comment by jakirkham** [comment]
Thanks Henry and James! 🙏 

Looks like the wheel builds are fixed 🥳

Conda still fails. Though that will depend on a release to address

**Comment by jameslamb** [comment]
closing, as this served its purpose for testing

**Pull request closed without merging**